js/mail.js

```diff
@@ -1194,6 +1185,35 @@ var Mail = {
 $(document).ready(function() {
 	Mail.UI.initializeInterface();
 
+	// Enables infinite scroll ability.
+	setInterval(function() {
+
+		// Element identifies the element we want to trigger our loading on.
+		//var element = $('.mail_message_summary:nth-last-child(1)');
+		var element = $('#load-more-mail-messages');
+		var outerHeight = element.outerHeight();
+		var offset = element.offset();
+		var scrollLength = $('#mail_messages').scrollTop();
+		var screenHeight = $(document).height();
+		var messageListHeight = scrollLength + offset.top + outerHeight;
+
+		if ((scrollLength + screenHeight) >= messageListHeight) {
+			Mail.UI.messageView.loadMessages('addMore');
+		}
+
+	}, 1000);
+
+	/**
+	 * Checks and loads any new messages and changes to the current folder.
+	 * @todo combine this with the checkForNotifications method.
+	 */
+	setInterval((function() {
+		// Begin the loop.
+		return function() {
+			Mail.UI.messageView.loadMessages();
+		};
+	})(), 10*1000);
```

Comment:
And failed. This new method works fine.

You're saying, yes, this works, but we prefer doing it our way, which at this point is only theory. If that's the case, keep your app. I'll fork. This bureaucracy is ridiculous.